src/components/Typography/Title/Title.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import styled from "styled-components";
+export type TitleColor = "default" | "muted";
+export type TitleSize = "xs" | "sm" | "md" | "lg" | "xl";
+export type TitleType = "product" | "brand";
+
+export interface TitleProps {
+  color?: TitleColor;
+  size?: TitleSize;
+  type?: TitleType;
+  children?: React.ReactNode;
+}
+
+/** The `title` component doesn't include an HTML H1 value by default to allow for maximum flexibility.
+ * Please remember to wrap your title text in an H tag to ensure correct HTML is being observed. */
+export const Title = ({
+  size = "md",
+  type = "product",
+  color = "default",
+  children,
+}: TitleProps) => (
+  <CuiTitle
```

Comment:
Should we pass the type like h1,h2 as a prop and assign it here
We could use as and then assign the type of header here
This could reduce the lines of codes and style it seamlessly

Reply:
Yeah I had tried to do this, but couldn’t find a way to convert the prop into an html tag (ie. <h1>) outside of using a long switch or something. Let me know if you have any ideas.

Reply:
lets not pass type. Derive a component of the Title component, where title is a children of an h1 element.
Export multiple components where each has its own type.
If someone really wants an H1 or an H2 they will import the right component.

Passing props with a type will imply ifs in the implementation, which is not a good thing for readability.

Reply:
@fneves So you're good with the current implementation? Which is 
`<title><h1>Main heading</h1></title>`?

Reply:
@gjones @fneves 
What I meant was adding a prop named `type` with values ['h1','h2','h3','h4','h5','h6"] and using it like
```

  <CuiTitle
    color={color}
    size={size}
    as={type}
  >
    {children}
  </CuiTitle>
````

This would result in <h1>children</h1>